open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/registration/AccessServiceDescription.java

```diff
@@ -141,12 +141,6 @@
                                       "governance-program",
                                       "Manage the governance program",
                                       "https://egeria.odpi.org/open-metadata-implementation/access-services/governance-program/"),
-    INFORMATION_VIEW_OMAS            (1014,
```

Comment:
Should not remove -

Reply:
Ok, can reinstate - but for my education can you clarify why as we've removed the OMAS? There should be no code left that tries to register using this? Is it to avoid any re-use?

Reply:
added back

> Also need to update ContentOrganization.md in top level directory

I didn't notice discrepancies specific to the removals I've made. were you referring to any entry in particular?

Reply:
I was imagining that the governance daemon connectors would have gone but I see there is still the open lineage connector :)